runtime/common/src/weights/pallet_democracy.rs

```diff
@@ -274,10 +287,68 @@ impl<T: frame_system::Config> pallet_democracy::WeightInfo for SubstrateWeight<T
 	// Proof: Democracy VotingOf (max_values: None, max_size: Some(3795), added: 6270, mode: MaxEncodedLen)
 	/// The range of component `r` is `[1, 100]`.
 	fn remove_other_vote(r: u32, ) -> Weight {
-		Weight::from_ref_time(15_709_616 as u64)
-			// Standard Error: 934
-			.saturating_add(Weight::from_ref_time(108_061 as u64).saturating_mul(r as u64))
+		Weight::from_ref_time(20_145_038 as u64)
+			// Standard Error: 2_964
+			.saturating_add(Weight::from_ref_time(132_196 as u64).saturating_mul(r as u64))
 			.saturating_add(T::DbWeight::get().reads(2 as u64))
 			.saturating_add(T::DbWeight::get().writes(2 as u64))
 	}
+	// Storage: Democracy NextExternal (r:1 w:0)
+	// Proof: Democracy NextExternal (max_values: Some(1), max_size: Some(132), added: 627, mode: MaxEncodedLen)
+	// Storage: Preimage StatusFor (r:1 w:0)
+	// Proof: Preimage StatusFor (max_values: None, max_size: Some(91), added: 2566, mode: MaxEncodedLen)
+	// Storage: Democracy MetadataOf (r:0 w:1)
+	// Proof: Democracy MetadataOf (max_values: None, max_size: Some(53), added: 2528, mode: MaxEncodedLen)
+	fn set_external_metadata() -> Weight {
```

Comment:
We should make sure in the PR and in the 1.7.0 release notes to reference these new extrinsics.